.github/workflows/settings-1-update.yml

```diff
@@ -0,0 +1,95 @@
+name: Config Settings Update
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - config/settings.*
+  pull_request:
+    branches:
+      - main
+    paths:
+      - config/settings.*
+env:
+  CONFIG_SETTINGS_PATH: ./config/settings.json
+  CONFIG_SETTINGS_SCHEMA_PATH: ./config/settings.schema.json
+jobs:
```

Comment:
It's kinda circular that 'paths' is used to trigger the build, but the same paths are set as environment variables. Can the environment variables be used in `paths`?

Reply:
Unfortunately not - they need to be static. The environment variables are just so I don't somehow get the path wrong later :D